index.html

```diff
@@ -407,7 +407,7 @@ <h4 class="uppercase mb16">Blogs</h4>
 		{% for blog in site.data.blogs %}
 			<div class="col-md-3 col-sm-6">
 				<a href="{{blog.url}}" target="_blank">
-					<div class="image-tile outer-title text-center">
+					<div class="image-tile outer-title text-center" style="margin-bottom: 0px;">
```

Comment:
@Ploypiti Please don't use inline styling, it is not considered as a good practice. Please read more about it [here](https://www.thoughtco.com/avoid-inline-styles-for-css-3466846)